### ISSUE TITLE
Remember device cookie 

### ISSUE THREAD

**Issue by agronv** [twilio/authy-devise#115]
I noticed that the remember_device cookie isn't set to HttpOnly. 
```
def remember_device
    id = @resource.id
    cookies.signed[:remember_device] = {
      :value => {expires: Time.now.to_i, id: id}.to_json,
      :secure => !(Rails.env.test? || Rails.env.development?),
      :expires => resource_class.authy_remember_device.from_now
    }
end
```

Is there a particular reason why it is not set? I can submit a PR if needed

**Comment by philnash** [comment]
You know, I don't think there's a reason this isn't set. Happy to accept a PR about this.

One thing I'm trying to think of though, is whether this is likely to break anything for existing users. I can't see why anyone would be reading the value of this cookie from their client side, but I don't want to be wrong either.

**Comment by agronv** [comment]
Thanks! I think it's safe to assume that no one wants users to access cookies from client scripts. This is my first time contributing to something like this do I have access to push branches or to create a PR?

**Comment by philnash** [comment]
You are in luck as it's Hacktoberfest and the Internet is awash with descriptions of how to contribute to a project like this! 

The quick run down is that you need to:

* Fork the repo to your own account
* Clone that repo
* Open a new branch
* Make the changes (including tests if you think that would be useful)
* Make sure the tests run successfully
* Commit and push your branch to your fork of the project
* Open a pull request against this repo

[Digital Ocean have a good, in depth, guide to all of this](https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github). Or if you want the most fun way to learn how to commit to open source, check out the [Open Pixel Project and our TwilioQuest mission that walks you through making your first contribution](https://www.twilio.com/blog/learn-contribute-open-source).

![](https://s3.amazonaws.com/com.twilio.prod.twilio-docs/images/hnEqUIn6_9V7Yb18D-uRLQCcM8HOFdtCgC62I9keDZNlpm.width-500.png)

Let me know if I can help any other way.

**Comment by philnash** [comment]
This is released in version 2.0.0.
